src/frontend/src/app.html

```diff
@@ -186,16 +186,10 @@
 				width: 100%;
 				min-width: 1728px;
 
-				display: none;
+				display: block;
```

Comment:
Given we set it to `blobk`, is this property still required?

Reply:
right! no, it is not, removed